client/components/token-field/suggestions-list.jsx

```diff
@@ -1,8 +1,9 @@
 /**
  * External dependencies
  */
-var map = require( 'lodash/map' ),
-	React = require( 'react' ),
+import { map } from 'lodash';
+
```

Comment:
I did a lot of manual patch-up for this PR on account of figuring it was easier than making the codemod work exactly as I wanted. I made the choice to add a space between the `import` lines and the `var _ = require( … );` lines. It doesn't have to be that way though. What say you?